### ISSUE TITLE
Please support all SystemMouseCursors on macOS (and other platforms)

### ISSUE THREAD

**Issue by sma** [flutter/flutter#92894]
Instead of not showing anything if you try to set `SystemMouseCursors.resizeUpLeftDownRight` or `SystemMouseCursors.resizeUpRightDownLeft` on macOS, please provide custom cursors that match the other up/down/left/right cursors or please provide a way that I can do this in a cross-platform way.

It is my understanding that those cursors aren't available because the standard macOS [NSCursor](https://developer.apple.com/documentation/appkit/nscursor?language=objc) class doesn't provide them.

But Flutter should be cross-platform and therefore, all `SystemMouseCursors` should work on all desktop platforms.

**Comment by HansMuller** [comment]
CC @dkwingsmt

**Comment by dkwingsmt** [comment]
Yes. We would like to support custom cursors from images. Please track the issue at https://github.com/flutter/flutter/issues/89351 (we haven't started it yet, but probably will shortly.)

Once that is done, I agree that it would be nice to have these cursors polyfilled, but I'm not sure if Flutter would like to maintain it, since there's an extent of flexibility in terms of graphic design. Maybe a third-party library would be a better solution. 

Anyway, yes we need to get the custom cursors supported.

**Comment by gspencergoog** [comment]
Closing this as a duplicate of #89351 (Desktop Triage)
